### ISSUE TITLE
Make pewpew available in package tools such as brew, apt

### ISSUE THREAD

**Issue by anuarkaliyev23** [bengadbois/pewpew#56]
I'm not sure if it fits under guidelines, since it's not directly related to tool itself, but I've felt that I should file an issue
I've searched for pewpew in homebrew repository, but haven't found it.

I know, that pewpew is available as a [release](https://github.com/bengadbois/pewpew/releases/tag/v1.0.0), but I think it would be more convenient to upload it to package repositories, such as homebrew, apt, aur and others.

It's a great tool and it would be great if it's installation could be even more straightforward

**Comment by bengadbois** [comment]
Good idea. I might look into goreleaser, it seems popularly used for this. Also open to PRs, but I assume as the repo owner I may need to do some permissions things.
